container.te

```diff
@@ -667,6 +667,9 @@ optional_policy(`
 optional_policy(`
 	unconfined_domain_noaudit(spc_t)
 	domain_ptrace_all_domains(spc_t)
+	// This should eventually be in upstream policy.
```

Comment:
@rhatdan could you the implications of this? I just want to  make sure we can consume it at the right places.

Reply:
This is just making an unconfined domain unconfined.  spc_t is supposed to be allowed to do anything it wants on a system.  This allow rule allows spc_t processes to load/run bpf programs on any executable/process on the system, from an SELinux point of view.

Reply:
Sorry @rhatdan I was unclear. I literally meant the comment. The implications of the comment are not clear to me and I just want to ensure that we get this in normal RPMs in fedora/centos stream.

Reply:
Ok, I put the fix in two places.  It really belongs in selinux-policy, but I put it in container-selinux, so that I can get it out to RHEL/Centos/Fedora ... earlier.

Reply:
Thansk for the clarification. :+1: